app/models/participant.rb

```diff
@@ -141,6 +141,10 @@ def first_middle
     [first_name, middle_initial].join(' ')
   end
 
+  def can_be_destroyed?
+    appointments.map{|appointment| appointment.procedures}.flatten.detect{|procedure| !procedure.unstarted?}.nil?
```

Comment:
another way to do this would be `appointments.all?{|a| a.procedures.all?(&:unstarted?)}`